faq.md

```diff
@@ -15,6 +15,23 @@ The final text of the CRA can be found on [EUR-Lex][CRA] ([English HTML version]
 The CRA enters into force on December 11, 2024. Reporting obligations of manufacturers ([Article 14][]) start to apply on September 11, 2026.
 The notitifcation of conformity of assement bodies ([Chapter IV][]) start to apply on June 11, 2026. Everything else starts to apply on December, 11 2027.
 
+#### Q: What is in scope of the CRA?
+
+The following types of product are in scope of the CRA:
+
+- Hardware products (e.g. laptops, smart appliances, mobile phones, network equipment, CPUs, etc.)
+- Software products (e.g. operating systems, word processing, games or mobile apps, software libraries, etc.)
+- Remote data processing solutions for any of the above
```

Comment:
This last line could use an "(e.g. [..])", perhaps based on one of the CRA-team's presentations, because that terms means absolutely nothing to people not already familiar.

Reply:
or, to name something absolutely at random:
> (e.g. a webservice that powers the spell-checking functionality of a word processing product)

Reply:
Fixed in 2799fed